test/e2e/kuma_deploy_global_local_test.go

```diff
@@ -75,25 +70,13 @@ var _ = Describe("Test Local and Global", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(logs2).To(ContainSubstring("\"mode\":\"local\""))
 
-		// when
-		status, response := http_helper.HttpGet(c1.GetTesting(), global.GetGlobaStatusAPI(), nil)
-		// then
-		Expect(status).To(Equal(http.StatusOK))
-
-		// when
-		clustersStatus := poller.Clusters{}
-		_ = json.Unmarshal([]byte(response), &clustersStatus)
-		// then
-
-		found := false
-		for _, cluster := range clustersStatus {
-			if cluster.URL == local.GetHostAPI() {
-				Expect(cluster.Active).To(BeTrue())
-				found = true
-				break
-			}
-		}
-		Expect(found).To(BeTrue())
+		err = DemoClient()(c2)
+		Expect(err).ToNot(HaveOccurred())
 
+		Eventually(func() string {
+			output, err := k8s.RunKubectlAndGetOutputE(c1.GetTesting(), c1.GetKubectlOptions("kuma-test"), "get", "dataplanes")
+			Expect(err).ToNot(HaveOccurred())
+			return output
+		}, "5s", "500ms").Should(ContainSubstring("demo-client-"))
```

Comment:
Can we have this as another `It` section? I mean I prefer to keep checking the API for the status of the registrations.